src/Microsoft.VisualBasic.Core/tests/InteractionTests.cs

```diff
@@ -114,6 +135,95 @@ public static IEnumerable<object[]> IIf_TestData()
             yield return new object[] { false, 3, "str", "str" };
             yield return new object[] { true, 3, "str", 3 };
         }
+        
+        [Fact]
+        public void DeleteSetting()
+        {
+            Assert.Throws<ArgumentException>(() => Interaction.DeleteSetting(AppName: "", Section: null, Key: null));
+            // Not tested: valid arguments.
+        }
+
+        [Fact]
+        public void Environ_Index()
+        {
+            var pairs = GetEnvironmentVariables();
+            int n = Math.Min(pairs.Length, 255);
+
+            Assert.Throws<ArgumentException>(() => Interaction.Environ(0)).ToString();
```

Comment:
>.ToString() [](start = 74, length = 11)

What's up with these tostring calls?

Reply:
And below.

---
In reply to: [314891998](https://github.com/dotnet/corefx/pull/40365#discussion_r314891998) [](ancestors = 314891998)

Reply:
Exception.ToString() is called to verify the exception message is constructed successfully. Added a comment and '_ =' as we discussed.

---
In reply to: [314892290](https://github.com/dotnet/corefx/pull/40365#discussion_r314892290) [](ancestors = 314892290,314891998)